index.js

```diff
@@ -366,13 +370,14 @@ const fields = [
   { key: 'tap', over: sortObject },
   { key: 'oclif', over: sortObjectBy(undefined, true) },
   { key: 'resolutions', over: sortObject },
-  { key: 'dependencies', over: sortObject },
-  { key: 'devDependencies', over: sortObject },
+  { key: 'overrides', over: sortDependenciesLikeNpm },
```

Comment:
also introduces sorting for [the `'overrides'` key for npm Overrides](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#overrides)